app/styles/layout/_page-wrapper.scss

```diff
@@ -21,3 +21,11 @@
 .site-head.theme-bar ~ .page-wrapper {
 	margin-top: 60px;
 }
+
+.bfaa-template {
+	.with-fandom-app-smart-banner {
+		.page-wrapper {
```

Comment:
minor thing: .bfaa-template .with-fandom-app-smart-banner .page-wrapper instead of nesting